src/PrestaShopBundle/Form/Admin/Type/DateRangeLocalType.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShopBundle\Form\Admin\Type;
+
+use DateTime;
+use PrestaShopBundle\Form\FormCloner;
+use Symfony\Component\Form\CallbackTransformer;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class DateRangeLocalType extends DateRangeType
+{
+    /**
+     * @var TranslatorInterface
+     */
+    protected $translator;
+
+    /**
+     * @var FormCloner
+     */
+    protected $formCloner;
+
+    /**
+     * @var string
+     */
+    protected $dateFormatFull;
```

Comment:
As I understand any date format can be passed to constructor in theory, so the variable name should not specify what kind of format it is. It should be simply:

```suggestion
    protected $dateFormat;
```

Reply:
I agree we shouldn't let the `Language` field name influence the naming here, also be careful this format from the Language is maybe a technical value adapted to the expected format of `DateTime::format` but it doesn't mean it's always understandable by a human so in some case the format might not be a good candidate for the placeholder which is destined to the user (I don't have a better alternative though for now 🤷‍♂️)